judgments/views/detail.py

```diff
@@ -69,6 +69,13 @@ def get_best_pdf(request, judgment_uri):
 
 def detail(request, judgment_uri):
     judgment = get_published_judgment_by_uri(judgment_uri)
+
+    # If the judgment_uri which was requested isn't the canonical URI of the judgment, redirect the user
+    if judgment_uri != judgment.uri:
+        return HttpResponseRedirect(
+            reverse("detail", kwargs={"judgment_uri": judgment.uri})
+        )
+
     context = {}
```

Comment:
Looks good. Would we be able to add a test that shows when a user goes to, for example caselaw.....//google.com, they aren't redirected, to verify the bug is fixed?